examples/foomedical/src/pages/ChatsPage.tsx

```diff
@@ -0,0 +1,72 @@
+import { Divider, Stack, Table, Title } from '@mantine/core';
+import { showNotification } from '@mantine/notifications';
+import { formatGivenName, formatFamilyName, normalizeErrorString } from '@medplum/core';
+import { Patient, Practitioner, HumanName } from '@medplum/fhirtypes';
+import { Document, useMedplum, useMedplumProfile } from '@medplum/react';
+import { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { Loading } from '../components/Loading';
+import classes from './ChatsPage.module.css';
+import { IconCircleOff } from '@tabler/icons-react';
+
+export function Chats(): JSX.Element {
+  const medplum = useMedplum();
+  const profile = useMedplumProfile() as Patient;
+  const navigate = useNavigate();
+  const [practitioners, setPractitioners] = useState<Practitioner[]>();
+
+  useEffect(() => {
```

Comment:
we actually have a new hook called `useSearch` inside our `@medplum/react-hooks` library. It didn't exist when we first created FooMedical, but I think this could simplify this code